darwin/future/meta/objects/base.py

```diff
@@ -21,4 +21,43 @@ def __init__(
         self.meta_params = meta_params or {}
 
     def __str__(self) -> str:
+        class_name = self.__class__.__name__
+        if class_name == "Team":
+            return f"Team\n\
+- Team Name: {self._element.name}\n\
+- Team Slug: {self._element.slug}\n\
+- Team ID: {self._element.id}\n\
+- {len(self._element.members if self._element.members else [])} member(s)"
+
+        elif class_name == "TeamMember":
+            return f"Team Member\n\
+- Name: {self._element.first_name} {self._element.last_name}\n\
+- Role: {self._element.role.value}\n\
+- Email: {self._element.email}\n\
+- User ID: {self._element.user_id}"
+
+        elif class_name == "Dataset":
+            return f"Dataset\n\
+- Name: {self._element.name}\n\
+- Dataset Slug: {self._element.slug}\n\
+- Dataset ID: {self._element.id}\n\
+- Dataset Releases: {self._element.releases if self._element.releases else 'No releases'}"
+
+        elif class_name == "Workflow":
+            return f"Workflow\n\
+- Workflow Name: {self._element.name}\n\
+- Workflow ID: {self._element.id}\n\
+- Connected Dataset ID: {self._element.dataset.id}\n\
+- Conneted Dataset Name: {self._element.dataset.name}"
```

Comment:
I thought about checking for a disconnected workflow - But it seems that when we create a workflow we raise an error if it's disconnected, so it should never happen here